demos/PersonalizerTravelAgencyDemo/PersonalizerTravelAgencyDemo/Actions/actions.json

```diff
@@ -1,194 +1,194 @@
+    "Image": "elephant",
     "Layout": "B",
     "ToneFont": "B",
     "ButtonColor": "B",
     "Enabled": true
   },
   {
     "Id": 9,
-    "Image": "A",
+    "Image": "elephant",
     "Layout": "C",
     "ToneFont": "A",
     "ButtonColor": "A",
     "Enabled": true
   },
   {
     "Id": 10,
-    "Image": "A",
+    "Image": "elephant",
     "Layout": "C",
     "ToneFont": "A",
     "ButtonColor": "B",
     "Enabled": true
   },
   {
     "Id": 11,
-    "Image": "A",
+    "Image": "elephant",
     "Layout": "C",
     "ToneFont": "B",
     "ButtonColor": "A",
     "Enabled": true
   },
   {
     "Id": 12,
-    "Image": "A",
+    "Image": "elephant",
     "Layout": "C",
     "ToneFont": "B",
     "ButtonColor": "B",
-    "Enabled": true,
+    "Enabled": true
   },
   {
     "Id": 13,
-    "Image": "B",
+    "Image": "caribbean-placeholder",
     "Layout": "A",
     "ToneFont": "A",
     "ButtonColor": "A",
-    "Enabled": true,
+    "Enabled": true
   },
   {
     "Id": 14,
-    "Image": "B",
+    "Image": "caribbean-placeholder",
     "Layout": "A",
     "ToneFont": "A",
     "ButtonColor": "B",
-    "Enabled": true,
+    "Enabled": true
   },
   {
     "Id": 15,
-    "Image": "B",
+    "Image": "caribbean-placeholder",
     "Layout": "A",
     "ToneFont": "B",
     "ButtonColor": "A",
-    "Enabled": true,
+    "Enabled": true
   },
   {
     "Id": 16,
-    "Image": "B",
+    "Image": "caribbean-placeholder",
     "Layout": "A",
     "ToneFont": "B",
     "ButtonColor": "B",
-    "Enabled": true,
+    "Enabled": true
   },
   {
     "Id": 17,
-    "Image": "B",
+    "Image": "caribbean-placeholder",
     "Layout": "B",
     "ToneFont": "A",
     "ButtonColor": "A",
-    "Enabled": true,
+    "Enabled": true
   },
   {
     "Id": 18,
-    "Image": "B",
+    "Image": "caribbean-placeholder",
     "Layout": "B",
     "ToneFont": "A",
     "ButtonColor": "B",
-    "Enabled": true,
+    "Enabled": true
   },
   {
     "Id": 19,
-    "Image": "B",
+    "Image": "caribbean-placeholder",
     "Layout": "B",
     "ToneFont": "B",
     "ButtonColor": "A",
-    "Enabled": true,
+    "Enabled": true
   },
   {
     "Id": 20,
-    "Image": "B",
+    "Image": "caribbean-placeholder",
     "Layout": "B",
     "ToneFont": "B",
     "ButtonColor": "B",
-    "Enabled": true,
+    "Enabled": true
   },
   {
     "Id": 21,
-    "Image": "B",
+    "Image": "caribbean-placeholder",
     "Layout": "C",
     "ToneFont": "A",
     "ButtonColor": "A",
-    "Enabled": true,
+    "Enabled": true
   },
   {
     "Id": 22,
-    "Image": "B",
+    "Image": "caribbean-placeholder",
     "Layout": "C",
     "ToneFont": "A",
     "ButtonColor": "B",
-    "Enabled": true,
+    "Enabled": true
   },
   {
     "Id": 23,
-    "Image": "B",
+    "Image": "caribbean-placeholder",
     "Layout": "C",
     "ToneFont": "B",
     "ButtonColor": "A",
-    "Enabled": true,
+    "Enabled": true
   },
   {
     "Id": 24,
-    "Image": "B",
+    "Image": "caribbean-placeholder",

```

Comment:
It might be better to have the image be an object, to have both the name and a proper alternate text. This name will probably sound bad when read through the narrator.

Reply:
@kbrons Done!